Update helpers.ts

ashachaf opened the pull request:
adding support to gnisis safe and xDefi wallets
#1536 
#1579

zavelevsky reviewed (approved):
looks straightforward

ashachaf commented:
@zavelevsky 
- i've added support to xdefi and gnosis
- i've updated the version to 1.34.2 (last known stable version)
checks:
- there is no loading issue
- metamask is now visible on both edge cases (fix the issue identify in version 1.35)

please review
